### DISCUSSION TITLE
[@types/steam-user] doesn't export interface types properly.

### DISCUSSION THREAD

**Discussion by JesseD045** [DefinitelyTyped/DefinitelyTyped#70801] [Issues with a @types package] [answered]
The declaration files in @types/steam-user declare a bunch of interfaces, such as those in `steam-user/components/chatroom.d.ts` that don't get exported. This is because of the `export = SteamChatRoomClient;` line which exports the `SteamChatRoomClient` class as default and typescript will not export any of the other interfaces declared in that file.

[a comment by an automated account is not shown]

**Comment by joshuajeschek** [comment]
I have a rework planned, but I cannot give you an ETA (very busy right now, might take a while). I have noted your feedback, however. Do you have any suggestions on how to improve the current situation?

**Reply by JesseD045** [reply]
From what I can tell, the easiest solution would be to explicitly export all of the declared interfaces in files that have a default export. I might consider doing it myself and making a PR. If I do, I'll reference it in this discussion.

**Comment by nik-webdevelop** [comment]
@joshuajeschek I stumbled upon this yesterday, and it's a huge blocker for me. I literally cannot import any of the interfaces from the .d.ts. Is there a way it can be done somehow, or could you just explicitly export all the interfaces, as @JD06450 suggested, please?

**Reply by joshuajeschek** [reply]
Please feel free to implement the suggested solution yourself, as I am still too busy to do so myself. Seems like nothing has changed in the last 8 months :D

**Reply by JesseD045** [reply]
Hello, I can get a fix pushed within a few hours. I got busy with other things and forgot about this along the way.

**Comment by JesseD045** [comment, marked as answer]
PR #73257 fixes this, and as it just got merged, I will mark this as resolved.
